sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1538,6 +1585,32 @@ pi_result _pi_platform::populateDeviceCacheIfNeeded() {
           delete[] ZeSubdevices;
           return Result;
         }
+
+        // sub-sub-device
+        // get all the ordinals for the sub-sub-devices
+        pi_uint32 SubSubDevicesCount = 0;
+        std::vector<int> Ordinals;
+        std::vector<ze_command_queue_group_properties_t> AllQueueProperties;
+        Result = getCmdQueueOrdinals(PiSubDevice.get(), SubSubDevicesCount, Ordinals, AllQueueProperties);
```

Comment:
I would recommend inlining the `getCmdQueueOrdinals` so you don't need to pass properties around.

Reply:
will do.

Reply:
done.